### PR TITLE
Re-use an existing local repo (project command)

### LINKED ISSUE #7: Re-use an existing local repo (projects)

### PR DESCRIPTION
**Pull request by dpi**
Fixes #7

### CODE REVIEW

File: src/Commands/ProjectMergeRequest.php [hunk 1]
```diff
@@ -55,11 +68,12 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+        [$httpFactory, $httpAsyncClient] = $this->http($logger, $options->noHttpCache, []);
+        $httpClientBuilder = new Builder($httpAsyncClient, $httpFactory);
+        $gitlab = new GitlabClient($httpClientBuilder);
+        $gitlab->setUrl('https://git.drupalcode.org');
 
-        //path, name,
-        $project = $client->projects()->show(sprintf('project/%s', $options->project));
+        $project = $gitlab->projects()->show(sprintf('project/%s', $options->project));
         $projectId = $project['id'] ?? null;
         if (!$projectId) {
```
Comment (jibran) [inline src/Commands/ProjectMergeRequest.php]: 77 can become if condition.

File: src/Commands/ProjectMergeRequest.php [hunk 2]
```diff
@@ -109,7 +123,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         );
 
         $response = $io->choice(
-            'Please select merge request to checkout',
+            'Select merge request to checkout',
             $choices,
         );
         $mrId = array_search($response, $choices, true);
```
Comment (jibran) [inline src/Commands/ProjectMergeRequest.php:139]: Can `$mrId` be null here if we select illegal choice?

Comment (dpi) [reply src/Commands/ProjectMergeRequest.php:139]: choice question from symfony console wont continue until a valid `$choice` is entered.

File: src/Commands/ProjectMergeRequest.php (hunk 1 above)
Comment (dpi) [reply src/Commands/ProjectMergeRequest.php]: done

**Pull request merged**